CODEOWNERS

```
@@ -0,0 +1,5 @@
+## THIS SHOULD ALWAYS BE FIRST ##
+* @BitGo/btc-team
+
+.github @BitGo/btc-team @BitGo/infra @BitGo/velocity
+renovate.json @BitGo/btc-team @BitGo/infra @BitGo/velocity
```

Velocity does not need to own these files:

```suggestion
.github @BitGo/btc-team @BitGo/infra
renovate.json @BitGo/btc-team @BitGo/infra
```

---

adjusted.